client/boot/project/wordpress-com.js

```diff
@@ -32,19 +32,21 @@ const config = require( 'config' ),
 import { getSelectedSiteId, getSectionName } from 'state/ui/selectors';
 import { setNextLayoutFocus, activateNextLayoutFocus } from 'state/ui/layout-focus/actions';
 
-function renderLayout( reduxStore ) {
-	const Layout = require( 'controller' ).ReduxWrappedLayout;
+function renderLayoutPromise( reduxStore ) {
+	return require( 'controller' ).getReduxWrappedLayout( reduxStore ).then(
```

Comment:
I'm not sure. I looked into how `require.ensure` handles errors, and it looks like we actually aren't doing any error handling for any of our `require.ensure` calls across the app. Are there other failure cases we need to look at here?